cypress/integration/models/developer/applicationinventory/analysis.ts

```diff
@@ -228,18 +228,18 @@ export class Analysis extends Application {
             .closest(trTag)
             .within(() => {
                 cy.get(analysisColumn)
-                    .find("div > div")
+                    .find("div > div:nth-child(2)", { timeout: 10800000 }) // 3h
+                    .should("not.have.text", "Not started")
+                    .and("not.have.text", "Scheduled")
+                    .and("not.have.text", "In-progress")
```

Comment:
Text  can also be "failed" @abrugaro .
We should just check for ".should(have.text, "Completed")

Reply:
Also , please run this branch in jenkins for tests source_analysis.test.ts and binary_analysis.test.ts and check if these tests pass with these changes.

Reply:
About the first comment:

Text can be failed but that is expected in some tests, that's way I'm excluding the texts we don't use. The expected status is passed as a parameter to the function, and it can be also failed.
That's way I'm excluding these 3 statuses, because we don't use them.

About the second comment, yes, both tests pass with that change (RUN 171), but this doesn't solve the memory problem.

Reply:
Why we are using so many free-typed strings? Can we use more of a consts? BTW String is immutable and each such can create a new object. that could affect the memory as well

Reply:
I think this should be an enum instead of 3 different consts, right now the PR is in WIP as I try to fix the memory leak, that's why they're hard-writen strings, once is finished I will refactor the code before submiting the PR.

Reply:
Moved to enum.